gatsby-node.js

```diff
@@ -236,15 +252,17 @@ exports.createPages = async ({ graphql, actions }) => {
         })
       })
     })
-  })
-  result.data.allMeasuresJson.edges.forEach(({ node }) => {
-    createPage({
-      path: `/${node.electionDate}/referendum/${node.fields.slug}`,
-      component: path.resolve("src/templates/measure.js"),
-      context: {
-        id: node.id,
-        slug: node.fields.slug,
-      },
+    node.Referendums.forEach(referendum => {
+      createPage({
+        path:
+          "/" +
+          node.Date +
+          "/referendums/" +
+          referendum.Title.toLowerCase()
+            .split(" ")
+            .join("-"),
```

Comment:
Looks like you configured the slug plugin for this node, can we use fields.slug instead?

Reply:
Ah, I thought I got all of them! Guess not!